src/ui/public/scripting_languages/index.js

```diff
@@ -33,7 +34,10 @@ export function GetEnabledScriptingLanguagesProvider($http) {
     return $http.get(chrome.addBasePath('/api/kibana/scripts/languages'))
       .then((res) => res.data)
       .catch(() => {
-        toastNotifications.addDanger('Error getting available scripting languages from Elasticsearch');
+        const toastText = i18n.translate('common.ui.scriptingLanguages.errorFetchingToastDescription', {
```

Comment:
errorFetchingToastDescription => errorFetchingAvailableScriptingLanguagesToastDescription

Reply:
also I believe `toastText` is redundant.